tap_zoom/schemas/webinars.json

```diff
@@ -337,7 +564,7 @@
         "weekly_days": {
           "type": [
             "null",
-            "integer"
+            "string"
```

Comment:
Looking at their API, this field is listed as a string but the explanation discusses scheduling webinars specifically, which would be the POST functionality. Have we validated the functionality in case their documentation has copypasta?